### ISSUE TITLE
Реализовать класс `Dataset` и `DataLoader` для работы с готовым датасетом Sig53.

### ISSUE THREAD

**Issue by ripidipi** [ripidipi/demonstration-of-signal-classification#2]
- [ ] Создать `datasets/sig53_dataset.py` с классом:
  ```python
  class Sig53Dataset(torch.utils.data.Dataset):
      def __init__(self, root, train=True, transform=None): ...
      def __len__(self): ...
      def __getitem__(self, idx): ...


- [ ] В transforms/signal_transforms.py реализовать:

  - Normalize()
  
  - AddGaussianNoise(std_dev)
  
  - RandomFrequencyOffset(max_offset)
